SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -2735,6 +2735,41 @@ bool CFlowOutput::WriteVolume_Output(CConfig *config, unsigned long Iter, bool f
   return false || force_writing;
 }
 
+void CFlowOutput::AddCommonFVMOutputs(const CConfig *config) {
```

Comment:
👍 Special reason to not move `COORD-XYZ`  here?

Reply:
Order matters I think.

Reply:
yep it does ... it needs to be the first, at least for paraview output